app/code/Magento/Integration/Model/CustomerTokenService.php

```diff
@@ -48,24 +49,32 @@ class CustomerTokenService implements \Magento\Integration\Api\CustomerTokenServ
      */
     private $requestThrottler;
 
+    /**
+     * @var Magento\Framework\Event\ManagerInterface
+     */
+    private $eventManager;
+
     /**
      * Initialize service
      *
      * @param TokenModelFactory $tokenModelFactory
      * @param AccountManagementInterface $accountManagement
      * @param TokenCollectionFactory $tokenModelCollectionFactory
      * @param \Magento\Integration\Model\CredentialsValidator $validatorHelper
+     * @param \Magento\Framework\Event\ManagerInterface $eventManager
      */
     public function __construct(
         TokenModelFactory $tokenModelFactory,
         AccountManagementInterface $accountManagement,
         TokenCollectionFactory $tokenModelCollectionFactory,
-        CredentialsValidator $validatorHelper
+        CredentialsValidator $validatorHelper,
+        ManagerInterface $eventManager
```

Comment:
Please make in backward compatible way. Please refer to the guide https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/

Reply:
Let me use the Backward Сompatibility Policy.

Reply:
Regarding [Backward compatible development](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)

Adding a constructor parameter

Add a new optional parameter to the constructor at the end of the arguments list instead of modifying the constructor.

In the constructor body, if the new dependency is not provided (i.e. the value of the introduced argument is null), fetch the dependency using Magento\Framework\App\ObjectManager::getInstance().

So
~~~php
 CredentialsValidator $validatorHelper,
 ManagerInterface $eventManager = null
...
 $this->eventManager = $eventManager ? : ?: \Magento\Framework\App\ObjectManager::getInstance()->get(ManagerInterface::class); 
~~~

Reply:
Extactly. Thank you @0m3r :)